bke-runtime/gp-bke-runtime-minimal-application/values.yaml

```diff
@@ -11,6 +11,11 @@ nameOverride: ""
 # metadata.label.app.kubernetes.io/name: gp-bke-runtime-minimal-application
 fullnameOverride: ""
 
+## List of database connections used in the application
+# database:
+#   - name: "myappdb"
+#     type: mysql
```

Comment:
Verwenden wir den type aktuell schon?

Reply:
Ja im minimal-application:
```
                  {{- if eq $e.type "mysql" }}
                  key: mysql-password
```
Gefällt mir nicht. (Warum müssen Entwickler immer die Technologie in Schnittstellen leaken 👎 ) Eigentlich sollte das DB-Bundle den Key standardisiert ins Secret schreiben, habe aber keine einfache Möglichkeit gefunden das zu machen. Leider gibt es keine Referenzen zu anderen Secrets. Es gibt Operators, die Secrets synchronisieren, das ist mir derzeit zu schwergewichtig. Ich schau mir mal an ob man mit Kustomize nachträglich Anpassungen machen könnte.